src/components/Range/index.js

```diff
@@ -431,6 +431,7 @@ const Range = ({
           offsetTop={offsetTop}
           width={cursorWidth}
           grabbed={grabbedCursor !== undefined}
+          // eslint-disable-next-line react/no-array-index-key
           key={`cursor-${index}`}
```

Comment:
can't you use value instead of index ? As cursor value is always unique

Reply:
I thought about it but it will change on every cursor move. Which might not be ideal performance wise? 🤔